src/c20_server/data_extractor.py

```diff
@@ -0,0 +1,24 @@
+
+from collections import namedtuple
+
+Data = namedtuple('Data', ['folder_name', 'file_name', 'contents'])
```

Comment:
How about to have Data as a class, I mean as we did for Job types. Because I think it will be easier to read for whom not familiar with namedtuple library and its usage.

Reply:
I think it is cleaner to just have the `namedtuple`.  It's more "Pythonic", and it's less code.